### DISCUSSION TITLE
DEI in RSE

### DISCUSSION THREAD

**Discussion by likeajumprope** [RSEAA/RSEAA.github.io#62] [Discussion topics for #RSEAA2023]
At RSE.con in Swansea it was revealed that the "median RSE" is white, male and has a Computer Science background.

How can we promote DEI in RSE?

**Comment by rowlandm** [comment]
I gave this presentation earlier this year - [Mosbergen, Rowland (2023). Improving Diversity and Inclusion in the RSE Community. figshare. Presentation. https://doi.org/10.6084/m9.figshare.22214551 ](https://doi.org/10.6084/m9.figshare.22214551)
